### DISCUSSION TITLE
Rollback and Migrate command

### DISCUSSION THREAD

**Discussion by abbasudo** [laravel/framework#51574] [Q&A] [answered]
is there any command that runs `artisan migrate:rollback` then `artisan migrate`? I use these commands sequentially often when I'm developing. In fact, I think in most cases developers need to migrate after rolling back the last changes.

isn't it a good idea to add such a command if not present?

**Comment by henzeb** [comment, marked as answer]
It should be `migrate:refresh --step 1`

**Reply by abbasudo** [reply]
Thanks

**Reply by ghost** [reply]
@henzeb this differs from` php artisan migrate:rollback --step=1` ? The --step space vs equal part.

**Reply by henzeb** [reply]
no. there just are two ways of writing things I find a space more convenient to type.

**Reply by ghost** [reply]
Did not knew that. So they are equivalent. Thank you.
